### PR TITLE
Fix tabs

### PR DESCRIPTION
**Pull request by manuelbuil**
Signed-off-by: Manuel Buil <mbuil@suse.com>

#### Proposed Changes ####

Tabs are suddenly not working. The only thing I found missing, according to the docs [1], is `alternate_style: true` 

[1] https://squidfunk.github.io/mkdocs-material/reference/content-tabs/

#### Types of Changes ####

Bugfix to our docs

#### Verification ####

Check https://docs.rke2.io/install/network_options/ and see if tabs appear again

#### Linked Issues ####

Do I need an issue? There is not going to be real testing, or?

### CODE REVIEW

**Review by briandowns — APPROVED** [review]

**Review by brandond — APPROVED** [review]

File: mkdocs.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ markdown_extensions:
 - codehilite
 - admonition
 - pymdownx.tabbed
+    alternate_style: true
```
Comment (brandond) [inline mkdocs.yml:22]: This doesn't look quite like valid YAML, should it be:
```suggestion
- pymdownx.tabbed:
    alternate_style: true 
```

**Review by brandond — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (manuelbuil) [reply mkdocs.yml:22]: oh man, thanks for that! :facepalm:

**Review by brandond — APPROVED** [review]

**Comment by brandond** [comment]
Merging this on behalf of @manuelbuil

**Comment by manuelbuil** [comment]
yeee!!! Tabs are back :)

**Pull request merged**